src/Components/ImagesTable/ImagesTable.js

```diff
@@ -286,6 +286,25 @@ const ImagesTable = () => {
                 );
               })}
           </TableComposable>
+          <Toolbar>
+            <ToolbarContent>
+              <ToolbarItem
+                variant="pagination"
+                align={{ default: 'alignRight' }}
+              >
+                <Pagination
```

Comment:
Can you add `variant={PaginationVariant.bottom}` so the dropdowns are dropups.

Reply:
@jkozol Added! :)